spec/event_store_client/connection_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module EventStoreClient
+  RSpec.describe Connection do
+    let(:client) do
+      EventStoreClient::StoreAdapter::Api::Client.new(host: 'https://example.com', port: 8080)
+    end
+    let(:connection) { described_class.new }
+    let(:event) { SomethingHappened.new(data: { foo: 'bar' }, metadata: {}) }
+
+    describe '#publish' do
+      before do
+        allow_any_instance_of(EventStoreClient::StoreAdapter::Api::Client).
+          to receive(:append_to_stream)
+      end
+
+      it 'returns serialized events' do
+        serialized_event = connection.publish(stream: 'stream', events: [event]).first
+        expect(JSON.parse(serialized_event.data)).to include('foo' => 'bar')
+        expect(serialized_event.type).to eq('EventStoreClient::SomethingHappened')
+      end
+    end
+
+    describe '#read' do; end
```

Comment:
These empty blocks should contain pending test example - at least one - so it'll appear in every rspec report and it'll be clean that there are tests to be done yet.